### PR TITLE
Ahrefs adjustments

### PR DESCRIPTION
**Pull request by freeminder**
tags adjustments for ahrefs.com

### CODE REVIEW

**Comment by monicazmf** [comment]
@usiegj00 Can you review please

File: app/views/protips/index.html.haml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 - title protips_title
-- description protips_description
+-# - description protips_description
```
Comment (usiegj00) [inline app/views/protips/index.html.haml]: What's the purpose of these changes?

File: app/views/protips/index.html.haml [hunk 2]
```diff
@@ -13,10 +13,10 @@
   .container
     .clearfix
       .sm-col.sm-col.sm-col-12.md-col-8
-        -if first_page?
-          .purple{style: "border-bottom:solid 5px;"}
-            %h2.mt0.black=protips_heading
-            %p.clearfix.py1.font-lg.black=protips_description
+        -# -if first_page?
```
Comment (usiegj00) [inline app/views/protips/index.html.haml]: This too?

File: app/views/sessions/new.html.haml [hunk 3]
```diff
@@ -1,7 +1,7 @@
-- title "Sign in"
+- title "Sign in to Coderwall"
 
 .container
-  %h2
+  %h1
```
Comment (usiegj00) [inline app/views/sessions/new.html.haml]: Is this for visual design or for SEO? If SEO, can you add a comment and make sure it's not affecting the visual (or should it?). Post a screenshot of before/after to help us understand.

File: app/views/protips/index.html.haml (hunk 2 above)
Comment (usiegj00) [reply app/views/protips/index.html.haml]: Screenshots for before/after would help here too.

File: app/views/protips/index.html.haml (hunk 1 above)
Comment (monicazmf) [reply app/views/protips/index.html.haml]: @usiegj00 The changes submitted are for the task related to META tags:
https://cl.ly/d26be953e613:

Meta-description too short
Meta description mission or empty
H1 tag missing or empty
Multiple H1 tags
Title too short

cc @freeminder

Comment (freeminder) [reply app/views/protips/index.html.haml]: There was no description meta tag in the rendered html at all.
Screenshot from 2019-06-05 13-27-31
Screenshot from 2019-06-05 13-28-00

File: app/views/protips/index.html.haml (hunk 2 above)
Comment (freeminder) [reply app/views/protips/index.html.haml]: There was no h1 tag at all pages after the first one.
Screenshot from 2019-06-05 13-30-53
Screenshot from 2019-06-05 13-31-45
Screenshot from 2019-06-05 13-32-21

File: app/views/sessions/new.html.haml (hunk 3 above)
Comment (freeminder) [reply app/views/sessions/new.html.haml]: This is for SEO. Check the last screenshot why h1 tag should be there.
It has some visual affect, i.e. text became bigger.
Screenshot from 2019-06-05 13-33-47
Screenshot from 2019-06-05 13-34-19
Screenshot from 2019-06-05 13-39-09

**Comment by freeminder** [comment]
@usiegj00 These tags' adjustments fixed the issues at ahrefs.com listed in the first screenshot.
Image 2019-05-29 at 6 24 43 PM
image_2019_06_04T15_03_37_019Z

File: app/views/protips/index.html.haml (hunk 1 above)
Comment (usiegj00) [reply app/views/protips/index.html.haml]: But you left commented code in the view, but didn't say why you commented it. Imagine someone new comes to look at this, will they understand why there is a commented line right there? Why did you comment it? Either add the comment so someone new will understand what you were thinking or remove the comment completely. Does that make sense? Try and look at this from someone that didn't know what you were doing or why. Be a good friend to your other developer teammates. :-P

**Comment by monicazmf** [comment]
@freeminder Please move your PR to the right repo here: https://github.com/aluminumio/coderwall/projects/2

This is an old project no longer to be used. 

Thanks
